corehq/apps/style/static/style/ko/components/inline_edit.js

```diff
@@ -79,6 +79,7 @@ hqDefine('style/ko/components/inline_edit.js', function() {
                     return;
                 }
 
+                self.value(self.value().replace(/<\/?script>/g, ''));
```

Comment:
this isn't quite fool proof. you can still execute javascript without a script tag. fwiw ive used DOMPurify in cloudcare to guard against this: https://github.com/cure53/DOMPurify. it's not quite the same scenario, but you can customize it to not allow any html tags (which is what i think we want)